packages/paste-core/components/alert-dialog/stories/index.stories.tsx

```diff
@@ -24,6 +24,7 @@ export default {
   ],
 };
 
+// @TODO replace usage of "React.FC", "React.ReactNode", "React.ReactElement" with "StoryFn" from "@storybook/react";
```

Comment:
Leaving a note here for future work. Tempted to make the change, but I resisted. 😅

Reply:
suggestion (non-blocking): I encourage you to make a ticket in Jira and advocate for it at the next planning session